modules/utilities.js

```diff
@@ -26,7 +28,26 @@ const parseValueFromString = str => {
     return upper || str;
 };
 
+const createHeroImage = opts => {
```

Comment:
Not really sure why these are expressions rather than a straight function declaration, but following the existing style in this file.

Reply:
True, they could safely be turned into function statements – no real reason other than force of habit in declaring stuff...